pkgs/development/python-modules/pymc3/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchurl
+, buildPythonPackage
+, Theano
+, gfortran
+, openblas
+, numpy
+, scipy
+, matplotlib
+, h5py
+, pandas
+, patsy
+, tqdm
+, joblib
+}:
+
+buildPythonPackage rec {
+  pname = "pymc3";
+  version = "3.2";
+
+  propagatedBuildInputs = [
+    Theano
+    gfortran
+    openblas
+    numpy
+    scipy
+    matplotlib
+    # unzip
+    h5py
+    pandas
+    patsy
+    tqdm
+    joblib
+  ];
+
+  doCheck = false;
```

Comment:
Why? Leave a comment in the code.

Reply:
I made the tests work but they are quite computationally intensive and some of them fail. Maybe I can patch it so that only few specific tests are run.

Another issue is that the tests are being run in the install phase instead of the check phase. Is it normal?

Reply:
No, not really.

Reply:
I disabled the tests and left a note on this peculiarity.